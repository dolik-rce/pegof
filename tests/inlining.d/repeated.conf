input inlining.d/repeated.peg
optimize inline
