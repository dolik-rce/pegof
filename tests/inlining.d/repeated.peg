X <- A+ (B C)* (D (E F))+ ((G H)?)?

A <- <"A"> { printf("%s\n", $1); }

B <- <"B"> { printf("%s\n", $1); }

C <- <"C"> { printf("%s\n", $1); }

D <- <"D"> { printf("%s\n", $1); }

E <- <"E"> { printf("%s\n", $1); }

F <- <"F"> { printf("%s\n", $1); }

G <- <"G"> { printf("%s\n", $1); }

H <- <"H"> { printf("%s\n", $1); }
