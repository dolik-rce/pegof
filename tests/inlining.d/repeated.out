X <- A+ (B C)* (D (E F))+ (((<"G"> { printf("%s\n", $1); }) (<"H"> { printf("%s\n", $2); }))?)?

A <- <"A"> { printf("%s\n", $1); }

B <- <"B"> { printf("%s\n", $1); }

C <- <"C"> { printf("%s\n", $1); }

D <- <"D"> { printf("%s\n", $1); }

E <- <"E"> { printf("%s\n", $1); }

F <- <"F"> { printf("%s\n", $1); }
